druid-derive/src/lens.rs

```diff
@@ -104,12 +104,30 @@ fn derive_struct(input: &syn::DeriveInput) -> Result<proc_macro2::TokenStream, s
         let field_ty = &f.ty;
 
         quote! {
-            impl #impl_generics druid::Lens<#ty#ty_generics, #field_ty> for #twizzled_name::#field_name #where_clause {
-                fn with<#val_ty_par, #func_ty_par: FnOnce(&#field_ty) -> #val_ty_par>(&self, data: &#ty#ty_generics, f: #func_ty_par) -> #val_ty_par {
+            impl #impl_generics druid::Lens<
```

Comment:
Are these just formatting changes? Did rustfmt do it? I wonder why

Reply:
Yes, they are only formatting changes; and rustfmt did not do it; I did it so I could read more easily as more variables were used